include/config/libinput.h

```diff
@@ -7,6 +7,9 @@
 #include <wayland-server-core.h>
 
 enum device_type {
+	LAB_LIBINPUT_DEVICE_NONE = 0,
+
+	// FIXME: Rename the entries below with a LAB_LIBINPUT_ prefix
```

Comment:
comment could be ... LAB_LIBINPUT_DEVICE_ prefix ...

Reply:
Overtaken by events. Have already sorted out the enums.